.gitattributes

```
@@ -1,3 +1,4 @@
 Cargo.lock linguist-generated=true
 /.gitlab-ci.yml filter=ci-prettier
 /scripts/ci/gitlab/pipeline/*.yml filter=ci-prettier
+frame/*/src/weights.rs linguist-generated=true
```

```suggestion
frame/**/src/weights.rs linguist-generated=true
```
There are two testing/examples pallets as well.